modules/calib3d/src/fisheye.cpp

```diff
@@ -1452,8 +1452,13 @@ void cv::internal::EstimateUncertainties(InputArrayOfArrays objectPoints, InputA
     CV_Assert(!omc.empty() && omc.type() == CV_64FC3);
     CV_Assert(!Tc.empty() && Tc.type() == CV_64FC3);
 
-    Mat ex((int)(objectPoints.getMat(0).total() * objectPoints.total()), 1, CV_64FC2);
-
+    int total_ex = 0;
+    for (int image_idx = 0; image_idx < (int)objectPoints.total(); ++image_idx)
+    {
+        total_ex += objectPoints.getMat(image_idx).total();
```

Comment:
To avoid the warning, change 

`total_ex += objectPoints.getMat(image_idx).total();`

to

`total_ex += (int)objectPoints.getMat(image_idx).total();`